### DISCUSSION TITLE
Username claim

### DISCUSSION THREAD

**Discussion by jathinyattapu** [community/community#112584] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

i need to change username. but already created by me in past. but i forgot all the details. but i need to claim the userid.

https://github.com/jathin/
i need to get this. please helpme.

**Comment by abdalla-am** [comment]
go to settings --> account --> change username
![Screenshot (220)](https://github.com/community/community/assets/112353203/80c3a543-4ca9-4d31-8244-e1241ac65f8e)

hope you find my comment helpful

**Comment by iamdulanga** [comment]
**I think you should contact github support..**

If you want that usename without delete that account you made in past, you have no option without reclaim it. to do it you have to login your account one way or another. if you really can't access. contacting github help is the best option.. if you able to delete the older account. you can change the username by **change username** option in the **account** section in **settings**.. 

_i hope you'll fix it.._

**Comment by CodeByAshuu** [comment]
Sure, here are the points summarized:

- Recover Access: Use the "Forgot password?" option on GitHub's login page to try to regain access to the old account associated with the desired username.

- Contact GitHub Support: If unable to recover access, reach out to GitHub Support for assistance. Provide necessary proof of ownership or identity to help regain access to the account.

- Consider Alternatives: If unable to recover the old account, consider choosing a different username for your current GitHub profile. Keep in mind that usernames are unique, and reclaiming an existing one may require proving ownership.

<i>I hope you'll fix it..</i>

**Comment by closebridge** [comment]
I do believe that GitHub is no longer accepting username requests, but you could try resetting your login information using the "Forget Password" feature

Good luck!
